internal/plugin/connectors/tcp/proxy_service.go

```diff
@@ -114,7 +115,8 @@ func (proxy *proxyService) handleConnection(clientConn net.Conn) error {
 	if err != nil {
 		return errors.Wrap(err, "failed on connect")
 	}
-
+	// immediately zeroize credentials after connecting
```

Comment:
nit: Use spacing liberally. I think this line needs a newline above it.

Reply:
Reinstated the newline above ✨